### PR TITLE
various cleanup of comments/names

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

File: ast/ast.go [hunk 1]
```diff
@@ -134,52 +134,52 @@ func (e *FieldCall) fieldExprNode() {}
 // ----------------------------------------------------------------------------
 // Procs
 
-// A proc is a node in the flowgraph that takes tuples in, processes them,
-// and produces tuples as output
+// A proc is a node in the flowgraph that takes records in, processes them,
+// and produces records as output
```
Comment (nwt) [inline ast/ast.go]: Nit: End sentence with a period.

File: pkg/zeek/column.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 )
 
-// Column defines the field name and type of a column in a tuple type
+// Column defines the field name and type of a column in a record type
```
Comment (nwt) [inline pkg/zeek/column.go]: New: End sentence with a period.

File: pkg/zson/descriptor.go [hunk 3]
```diff
@@ -13,9 +11,9 @@ type Resolver interface {
 	Lookup(td int) *Descriptor
 }
 
-// Descriptor describes the field names and types of a Tuple.
-// It is a list of the column descriptors along with a
-// map to do a fast lookup table of column index by field name
+// Descriptor as en entry for mapping small-integer descriptors to
```
Comment (nwt) [inline pkg/zson/descriptor.go]: s/as en/is an/

File: ast/ast.go [hunk 4]
```diff
@@ -226,7 +226,7 @@ type (
 	}
 )
 
-//XXX placeholder till we fix the client together with boom
+//XXX placeholder
```
Comment (mattnibs) [inline ast/ast.go]: maybe expand on what this a placeholder for? Shouldn't this be a nano.Ts?

File: pkg/zson/descriptor.go (hunk 3 above)
Comment (mattnibs) [inline pkg/zson/descriptor.go]:
```suggestion
// Descriptor is an entry for mapping small-integer descriptors to
```

**Pull request merged**